modules/compute/vm-instance/main.tf

```diff
@@ -133,6 +133,24 @@ resource "google_compute_instance" "compute_vm" {
     nic_type   = local.enable_gvnic ? "GVNIC" : null
   }
 
+  dynamic "network_interface" {
+    for_each = var.additional_network_interfaces
```

Comment:
I think it is a good practice to keep parameter names whenever possible. In this case, you are effectively renaming network interface with "additional_network_interfaces". I would suggest you keep the same name. In fact, why not keep it the same variable?

Reply:
Variable name was updated. 
Terraform doesn't appear to support setting blocks as variables: https://discuss.hashicorp.com/t/use-block-variables/5527/3